content-src/asrouter/templates/StartupOverlay/_StartupOverlay.scss

```diff
@@ -1,343 +0,0 @@
-      box-shadow: 0 0 0 3px $email-input-invalid;
-    }
-  }
-
-  .form-header {
-    font-size: 22px;
-    margin: 15px auto;
-  }
-
-  .form-header .sub-header {
-    font-size: 14px;
-    margin-top: 4px;
-    display: block;
-  }
-
-  button {
-    display: block;
-    cursor: pointer;
-    margin: 10px auto 0;
-  }
-
-  .continue-button {
-    font-size: 18px;
-    height: 43px;
-    width: 250px;
-    padding: 8px 0;
-    border: 0;
-    border-radius: 4px;
-    color: $white;
-    background-color: $blue-60;
-    transition: background-color 150ms;
-
-    &:not([disabled]):active,
-    &:not([disabled]):hover {
-      background: $blue-70;
-      border-color: $blue-80;
-    }
-  }
-
-  .skip-button {
-    font-size: 13px;
-    margin-top: 20px;
-    margin-bottom: 20px;
-    background-color: $grey-10;
-    color: $blue-50;
-    border: 1px solid $blue-50;
-    border-radius: 2px;
-    min-height: 24px;
-    padding: 5px 10px;
-    transition: background-color 150ms, color 150ms, border-color 150ms;
-
-    &[disabled] {
-      background-color: $grey-20;
-      border-color: $grey-40;
-      color: $grey-50;
-      cursor: default;
-      opacity: 0.5;
-    }
-
-    &:not([disabled]):hover {
-      background-color: $blue-50;
-      border-color: $blue-60;
-      color: $white;
-    }
-  }
-}
-
-.firstrun-left-divider {
-  position: relative;
-  float: inline-start;
-  clear: both;
-  width: 435px;
-
-  @media screen and (max-width: 825px) {
-    width: 400px;
-  }
-
-  @media screen and (max-width: 790px) {
-    margin: auto;
-    float: none;
-    width: 352px;
-    text-align: center;
-  }
-}
-
-.firstrun-content {
-  line-height: 1.5;
-  margin-bottom: 48px;
-  max-width: 352px;
-  background: url('#{$image-path}sync-devices.svg') bottom center no-repeat;
-  padding-bottom: 210px;
-}
-
-a.firstrun-link { // sass-lint:disable-line no-qualifying-elements
-  color: $white;
-  display: block;
-  text-decoration: underline;
-
-  &:hover,
-  &:active,
-  &:focus {
-    color: $white;
-  }
-}
-
-.firstrun-title {
-  background: url('chrome://branding/content/about-logo.png') top left no-repeat;
-  background-size: 90px 90px;
-  margin: 40px 0 10px;
-  padding-top: 110px;
-
-  @media screen and (max-width: 790px) {
-    background: url('chrome://branding/content/about-logo.png') top center no-repeat;
-    background-size: 90px 90px;
-  }
-
-  &:dir(rtl) {
-    background-position: top right;
-  }
-}
-
-.fxaccounts-container {
-  position: absolute;
-  bottom: 0;
-  right: 0;
-  top: 0;
-  left: 0;
-  color: $white;
-  height: 515px;
-  margin: auto;
-  width: 819px;
-  z-index: 10;
-  transition: opacity 0.3s;
-  opacity: 0;
-
-  @media screen and (max-width: 825px) {
-    width: 784px;
-  }
-
-  @media screen and (max-width: 790px) {
-    width: auto;
-    height: 100%;
-  }
-}
-
-.firstrun-title,
-.firstrun-content,
-.firstrun-link {
-  opacity: 0;
-  transform: translateY(-5px);
-}
-
-.error {
-  display: none;
-}
-
-.error.active {
```

Comment:
Looks like we are reusing these styles in trailhead interrupts. Invalid email error shows incorrect without these.